src/metatrain/utils/loss.py

```diff
@@ -149,6 +149,11 @@ def __call__(
             block_2 = targets_tensor_map.block(key)
             values_1 = block_1.values
             values_2 = block_2.values
+            # skip in case of empty blocks, which can occur for atomic basis spherical
+            # targets, for instance
+            if values_1.shape[0] == 0:
+                assert values_2.shape[0] == 0
+                continue
```

Comment:
This wouldn't be necessary if we decided to go for adding the empty blocks

Reply:
No longer needed